webapps/ab-testing-design/body.html

```diff
@@ -0,0 +1,538 @@
+
+<script src="/plugins/ab-testing/resource/js/visualisation/statistics.js" type="text/javascript"></script>
+<script src="/plugins/ab-testing/resource/js/visualisation/chart.js" type="text/javascript"></script>
+
+<title>Sample size estimator</title>
+
+<header class="navbar bd-navbar bg-light background no-margin">
+    <a class="navbar-brand" id=href="#">
+        <img src="https://www.dataiku.com/static/img/theme/logo/page.png" width="180" height="64" alt=""> </img>
+    </a>
+    <div class="d-flex justify-content-between">
+        <div>
+            <h2>Sample size estimator</h2>
+        </div>
+    </div>
+</header>
+
+<body>
+    <div class="flex-container" ng-app="abApp" ng-controller="SizeController">
+        <div class="background">
+            <div class="container white-background title-padding">
+                <div class="row">
+                    <div class="col-xl">
+                        <h1>Frequentist AB Testing - Sample size estimator </h1>
+
+                        <a class="anchorjs-link " href="#overview" aria-label="Anchor" data-anchorjs-icon="#"
+                            style="padding-left: 0.375em;"></a>
+                        <h2>Compare two proportions : conversion rate, click through rate.</h2>
+                        <p class="explanations">
+                            The average conversion rates of two independent samples A and B are compared. A is the
+                            control
+                            group
+                            while B is the experiment group. We assume that the two samples follow binomial
+                            distributions.
+
+                        </p>
+                    </div>
+                </div>
+                <div class="row border-top-custom">
+                    <div class="col-3 border-custom">
+                        <form id="form_data">
+                            <table class="table_fields">
+                                <tr>
+                                    <td class="field">
+                                        <label for="bcr">Baseline success rate (%)
+                                            <button id="info_bcr" type="button" class="btn btn-link"
+                                                data-toggle="popover" title="Baseline success rate" data-content="The baseline success rate defines the current success rate of
+                                                the campaign or the page.
+                                                This is the success rate of the control group A.
+                                                It equals to the number of successes divided by the reach.">
+                                                [?] </button>
+                                        </label>
+                                        <input type="number" name="bcr" placeholder="BCR" id="bcr" ng-model="bcr"
```

Comment:
why not put "enter a value between 0 and 100" as placeholder (you can make the placeholder red also) to make the UI lighter.

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/5de4b917247412051027057edfc773646bf5bdd5